testutil/ciao-down/cloudinit.go

```diff
@@ -196,6 +196,9 @@ runcmd:
  - {{download . "https://download.fedoraproject.org/pub/fedora/linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-24-1.2.x86_64.qcow2" (printf "/home/%%s/local/Fedora-Cloud-Base-24-1.2.x86_64.qcow2" .User)}}
  - {{template "CHECK" .}}
 
+ - curl -X PUT -d "Downloading xenial-server-cloudimg-amd64-disk1.img" 10.0.2.2:{{.HTTPServerPort}}
+ - {{download . "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img" (printf "/home/%%s/local/xenial-server-cloudimg-amd64-disk1.img" .User)}}
+
```

Comment:
We could drop the clear image but this is quick to upload so there's not much point.  Dropping Fedora would require changing some of the BAT tests but it would speed up singlevm startup by about 15 seconds.  @kaccardi what do you think?  Shall we ditch fedora?